lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -34,6 +36,8 @@ def commands
       "db_connect"    => "Connect to an existing database",
```

Comment:
Minor: a bunch of the language has changed to be more about data service than database. `db_connect` and `db_disconnect` descriptions should probably be updated